### PR TITLE
added a sweeping classroom chore

### PR DESCRIPTION
**Pull request by Ainatiana**
Hello teammates!

I have changed something in order to make things look better and new.

thank you,
Ainatiana,

### CODE REVIEW

File: sweeping-classroom.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# sweeping classroom
```
Comment (Tiaraha-funny) [inline sweeping-classroom.md]: Hihihi!! 

Good job man, I don't have any comment, thanks for that.
bye Peta.

**Review by Tiaraha-funny — APPROVED** [review]

File: sweeping-classroom.md (hunk 1 above)
Comment (Tiaraha-funny) [reply sweeping-classroom.md]: Hey aina did you get it? that's it for today!!!
see you next time!! 
your best friend who is never forget you!!!

Comment (Ainatiana) [reply sweeping-classroom.md]: Thanks Petha

Comment (ndorfin) [reply sweeping-classroom.md]: Hmmm, that `s` needs to go!

Comment (Ainatiana) [reply sweeping-classroom.md]: Please could you just review it @Tiaraha-funny second time?

Comment (Ainatiana) [reply sweeping-classroom.md]: https://www.google.com/url?sa=i&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwjWzKLhsZznAhXHUhUIHS4DCqQQMwhmKAYwBg&url=https%3A%2F%2Fgiphy.com%2Fgifs%2Fdrink-please-puppy-dog-eyes-TKY4CQ3Xk76qQ&psig=AOvVaw05YVIgrjCIjvqp8pt3ajPf&ust=1579960527790400&ictx=3&uact=3

**Pull request merged**